core/src/main/scala/zio/config/ConfigSource.scala

```diff
@@ -1,35 +1,56 @@
 package zio.config
 
+import java.io.{ File, FileInputStream }
 import java.{ util => ju }
-import zio.UIO
+
+import zio.config.PropertyTree.unflatten
+import zio.{ Task, UIO, ZIO }
 
 import scala.collection.JavaConverters._
 import scala.collection.immutable.Nil
-import zio.config.PropertyTree.unflatten
-import zio.ZIO
-import java.io.FileInputStream
-import java.io.File
-import zio.Task
 
 final case class ConfigSource[K, V](
   getConfigValue: Vector[K] => PropertyTree[K, V],
   sourceDescription: List[String]
 ) { self =>
-  final def orElse(that: => ConfigSource[K, V]): ConfigSource[K, V] =
+  def orElse(that: => ConfigSource[K, V]): ConfigSource[K, V] =
     ConfigSource(
       k => getConfigValue(k).getOrElse(that.getConfigValue(k)),
       if (self.sourceDescription == that.sourceDescription) self.sourceDescription
       else self.sourceDescription ++ that.sourceDescription
     )
 
-  final def <>(that: => ConfigSource[K, V]): ConfigSource[K, V] =
+  def <>(that: => ConfigSource[K, V]): ConfigSource[K, V] =
```

Comment:
All of our methods are final. As a standard

Reply:
`ConfigSource` class is final, so methods are already final.